tools/merger/data/ci/plugin/job.yml.template

```diff
@@ -33,7 +33,11 @@
           name: create test databases
           command: cd core && ./bin/create_test_databases
 {{{custom_steps}}}
+{{^custom_test}}
       - run:
           name: test
           command: cd {{{relative_path}}} && ./node_modules/.bin/jest  --ci --passWithNoTests --runInBand
-{{custom_test}}
+{{/custom_test}}
+{{#custom_test}}
+{{{custom_test}}}
+{{/custom_test}}
```

Comment:
I'm not totally sure if this change is necessary, since I don't see where `custom_test` is being set, but seems ok.

Reply:
`{{custom_test}}` ends up being sourced from any `test.yml.template` file within a plugin's folder.  That's how the override for the redshift test is getting built up.  The / and # are if blocks in mustache... so this is roughly saying if there is not a custom test use this comand here, otherwise use the one from the other file